src-common/main/scala/uk/gov/hmrc/audit/handler/DatastreamHandler.scala

```diff
@@ -30,41 +30,47 @@ class DatastreamHandler(
   host    : String,
   port    : Integer,
   path    : String,
-  wsClient: WSClient
+  wsClient: WSClient,
+  metrics: DatastreamMetrics,
+  logger : Logger = LoggerFactory.getLogger(getClass)
 ) extends HttpHandler(
   endpointUrl = new URL(s"$scheme://$host:$port$path"),
   wsClient    = wsClient
 ) with AuditHandler {
 
-  private val logger: Logger = LoggerFactory.getLogger(getClass)
-
   override def sendEvent(event: JsValue)(implicit ec: ExecutionContext): Future[HandlerResult] =
     sendEvent(event, retryIfMalformed = true)
 
   private def sendEvent(event: JsValue, retryIfMalformed: Boolean)(implicit ec: ExecutionContext): Future[HandlerResult] =
     sendHttpRequest(event).flatMap {
       case HttpResult.Response(status) =>
         Future.successful(status match {
-          case 204 => Success
-          case 400 => logger.warn("Malformed request rejected by Datastream")
-                      Rejected
-          case 413 => logger.warn("Too large request rejected by Datastream")
-                      Rejected
-          case _   => logger.error(s"Unknown return value $status")
-                      Failure
+          case status if 200 <= status && status <= 299 =>
```

Comment:
you could use
`case status if play.api.http.Status.isSuccessful(status) =>`

Reply:
Works for me :)